Wrappers/Python/cil/optimisation/functions/L1Norm.py

```diff
@@ -285,7 +284,7 @@ def convex_conjugate(self,x):
         if np.any(x.abs() > self.weight): # This handles weight being zero problems
             return np.inf
         # Avoid division by the weight
-        if np.any((x.abs() - self.weight) > 1e-8*self.weight):
+        if np.any((x.abs() - self.weight) > 0):
```

Comment:
Looking at this, isn't this check now equivalent to the first one? I.e. we could skip it and go straight to the `else:` condition.